<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoardRight</class>
 <widget class="QMainWindow" name="BoardRight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BOARD RIGHT</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/board_right_BKG.jpg);</string>
   </property>
   <widget class="SingleButton" name="CONDisy_widget" native="true">
    <property name="geometry">
     <rect>
      <x>803</x>
      <y>29</y>
      <width>78</width>
      <height>78</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="DESDisy_widget" native="true">
    <property name="geometry">
     <rect>
      <x>916</x>
      <y>29</y>
      <width>78</width>
      <height>78</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="calientapies" native="true">
    <property name="geometry">
     <rect>
      <x>803</x>
      <y>152</y>
      <width>78</width>
      <height>78</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="frenoRetencion" native="true">
    <property name="geometry">
     <rect>
      <x>916</x>
      <y>152</y>
      <width>78</width>
      <height>78</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="desacople" native="true">
    <property name="geometry">
     <rect>
      <x>916</x>
      <y>498</y>
      <width>78</width>
      <height>78</height>
     </rect>
    </property>
   </widget>
   <widget class="LlaveModoConduccion" name="modoConduccion" native="true">
    <property name="geometry">
     <rect>
      <x>787</x>
      <y>496</y>
      <width>110</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="Manometer" name="manometer" native="true">
    <property name="geometry">
     <rect>
      <x>807</x>
      <y>273</y>
      <width>188</width>
      <height>182</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>188</width>
      <height>182</height>
     </size>
    </property>
   </widget>
   <widget class="SicasMac" name="sicasmac" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>627</width>
      <height>482</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SingleButton</class>
   <extends>QWidget</extends>
   <header>singlebutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Manometer</class>
   <extends>QWidget</extends>
   <header>manometer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SicasMac</class>
   <extends>QWidget</extends>
   <header>sicasmac.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LlaveModoConduccion</class>
   <extends>QWidget</extends>
   <header>llavemodoconduccion.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
